tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -1159,10 +1164,6 @@ def test_keras_autolog_infers_model_signature_correctly_with_tf_dataset(fashion_
 
 
 @pytest.mark.large
-@pytest.mark.skipif(
-    Version(tf.__version__) < Version("2.6.0"),
-    reason="TensorFlow autologging is not used for vanilla Keras models in Keras < 2.6.0",
-)
```

Comment:
See https://github.com/mlflow/mlflow/pull/5652/files#r846732514 (and ditto below)